judgels-frontends/raphael/src/routes/uriel/contests/single/clarifications/ContestClarificationsPage/ContestClarificationsPage.tsx

```diff
@@ -102,6 +104,13 @@ class ContestClarificationsPage extends React.Component<
     };
     return <ContestClarificationCreateDialog {...props} />;
   };
+
+  private renderCreateAnswerDialog = () => {
```

Comment:
You need to check whether the current user is allowed to answer, before rendering anything.

E.g. add `isAllowedToAnswerClarification` to `ContestConfig` (backend-side)